README.md

```diff
@@ -16,4 +16,22 @@ Guitar Dashboard is written in Typescript using VS Code. Make all code changes i
 5. Browse to http://localhost:10001/
 6. Edit the src/*.ts, index.html and gtr-cof.css files.
 7. Compile.
-8. Commit, push to GitHub and create a pull request :)
\ No newline at end of file
+8. Commit, push to GitHub and create a pull request :)
+
+
+## Developing without VS Code
+
+First, make sure you have TypeScript installed. If not, `npm install -g typescript` will do the trick.
+
+1. Clone the repo and go into it
+2. Run `npm install`
+3. Open a shell and run `tsc --watch` so that the sources are always rebuilt automatically on source changes
```

Comment:
could be `npx tsc --watch` to run project's typescript instead of installing typescript globally

Reply:
I updated this here to simply `npm dev`: https://github.com/mikehadlow/gtr-cof/pull/30/files#diff-04c6e90faac2675aa89e2176d2eec7d8R14